schemas/rum/_common-schema.json

```diff
@@ -149,6 +149,12 @@
           },
           "readOnly": true
         },
+        "effective_type": {
+          "type": "string",
+          "description": "Effective type of the network connection",
+          "enum": ["slow_2g", "2g", "3g", "4g"],
+          "readOnly": true
+        },
```

Comment:
On Mobile, we're using the `cellular.technology` attribute to display the type of mobile network connection. What's the purpose of the additional "effective_type" ? Also don't you also need non mobile connection types (ADSL, Fiber, …)?

Reply:
Looking at what is collected in `cellular.technology`, I don't think we have an API to retrieve something similar.
Effective type gives an indication of the speed of the network, the listed values are the ones provided by the browser API.
cf [NetworkInformation](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation) for more details on what is available on the browser side.

Reply:
Ok, the naming is a bit ambiguous, maybe the documentation could reflect better the meaning, e.g.: 
> Bucket describing the speed and bandwidth of the network connection based on measurements of mobile data connections

Reply:
Updated with:
> Cellular connection type reflecting the measured network performance

to stay close to the [mdn doc wording](https://developer.mozilla.org/en-US/docs/Glossary/Effective_connection_type)